rd235_libslirp/src/qemu2libslirp-bool.h

```diff
@@ -0,0 +1,8 @@
+#ifndef QEMU2SLIRPLIB_BOOL_H
+#define QEMU2SLIRPLIB_BOOL_H
+
+/* TODO: switch to stdbool.h (caution: storage size is different) */
+typedef int bool;
```

Comment:
`typedef uint8_t bool` would be more space-efficient, but I don't understand why you didn't want to use `stdbool`.

Reply:
There was conflict between `stdbool.h` (used by `main.c`) and `typedef int bool` defined in `qemu2libslrip.h`.

This PR enforces to use the `typedef int bool` across all files consistently.

Transition to `stdbool` would be safe, but I didn't test. (testing is welcome)